code/modules/reagents/chemistry/holder.dm

```diff
@@ -507,6 +507,7 @@
 				continue
 			if(methods)
 				r_to_send += reagent
+				reagent.on_transfer(target_atom, methods, transfer_amount * multiplier)
```

Comment:
Is this order correct? Down lower has on_transfer called AFTER expose_reagent / expose_multiple